### DISCUSSION TITLE
Please add these buttons/links

### DISCUSSION THREAD

**Discussion by jerxjac** [Nerwyn/universal-remote-card#102] [Ideas]
Can you please add  icons/links shown below?

  FilmPlus:
    source: com.guideplus.co
![filmplus](https://github.com/Nerwyn/android-tv-card/assets/38360412/4794e658-6357-4b48-a937-40b14dfaba24)

  npo start:
    source: nl.uitzendinggemist
![npo](https://github.com/Nerwyn/android-tv-card/assets/38360412/81bb9c79-f724-40a7-b7a5-dd60b35428dd)

  kijk:
    source: com.talpa.kijk
![kijk](https://github.com/Nerwyn/android-tv-card/assets/38360412/ac5daa87-e945-4f51-b082-ff47b024607e)

**Comment by Nerwyn** [comment]
Can you make a feature request issue so I don't forget? It's easy to lose track of discussions.
